site/docs/sql/intro.md

```diff
@@ -0,0 +1,306 @@
+---
+title: Hazelcast Jet SQL
+description: Introduction to Hazelcast Jet SQL features.
+---
+
+Hazelcast Jet allows you to create a Jet processing job using the
+familiar SQL language. It can execute distributed SQL statements over
+Hazelcast IMaps and external data sets.
+
+**Note:** _The service is in beta state. Behavior and API might change
+in future releases. Binary compatibility is not guaranteed between minor
+or patch releases._
+
+## Overview
+
+In the first release, Jet SQL supports the following features:
+
+- SQL queries over [Apache Kafka topics](kafka-connector.md) and files
+(local and remote) (TODO)
+- Joining Kafka or file data with local IMaps (enrichment)
+- Filtering and projection using [SQL
+expressions](https://docs.hazelcast.org/docs/{imdg-version}/manual/html-single/index.html#expressions)
+- Aggregating data from files using predefined [aggregate
+functions](basic-commands#aggregation-functions)
+- Receiving query results via Jet client (Java) or writing the results
+to an [IMap](imap-connector.md) in the Jet cluster
+- Running continuous (streaming) and batch queries, see
+[Job Management](job-management.md)
+
+These are some of the features on our roadmap:
+
+- Joins with arbitrary external data sources
+- Windowed aggregation of streaming data
+- JDBC
+
+## Installation
+
+You need the `hazelcast-jet-sql` module on your classpath. For
+Gradle or Maven, make sure to add the dependency:
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Gradle-->
+
+```groovy
+compile 'com.hazelcast.jet:hazelcast-jet-sql:{jet-version}'
+```
+
+<!--Maven-->
+
+```xml
+<dependency>
+    <groupId>com.hazelcast.jet</groupId>
+    <artifactId>hazelcast-jet-sql</artifactId>
+    <version>{jet-version}</version>
+</dependency>
+```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+If you're using the distribution package, make sure to move the
+`hazelcast-jet-sql-{jet-version}.jar` file from the `opt/` to the `lib/`
+directory.
+
+If you use other features from SQL such as the Kafka or File connectors,
+also add the `hazelcast-jet-kafka` or `hazelcast-jet-hadoop` modules in
+the same way.
+
+## Example
+
+In this example we'll show how to query Apache Kafka using SQL and
+stream the messages to an IMap.
+
+First, make sure you have Apache Kafka up and running. Please follow any
```

Comment:
Include commands to install Kafka. We need code that can be copy-pasted and works. Also, we want to control Kafka server versioning and ports.